frontend/app/src/scenes/Home/scenes/UpsertCategory/metaData.ts

```diff
@@ -0,0 +1,15 @@
+import { TranslationMetaData } from './interfaces';
+
+const translationMetaData: Map<string, TranslationMetaData> = new Map();
+translationMetaData.set('SI_LK', {
+  name: 'සිංහල',
+  placeholder: 'පරිවර්තනය',
```

Comment:
this should be the single translation of the title field's placeholder. 

the placeholder of the title field is: "ex: KG & Primary" 

add a good placeholder and translate it here

Reply:
sure

Reply:
or else you can add, 

"මාතෘකාව සඳහා සිංහල පරිවරතනය ඇතුලත් කරන්න"

Reply:
ok